package.json

```diff
@@ -48,12 +48,17 @@
     "jshint": "jshint --show-non-errors ."
   },
   "dependencies": {
-    "sax": "~1.2.1",
     "coa": "~1.0.1",
-    "js-yaml": "~3.7.0",
     "colors": "~1.1.2",
+    "css-select": "^1.3.0-rc0",
+    "css-select-base-adapter": "^0.1.0",
+    "css-tree": "1.0.0-alpha18",
+    "csso": "^3.0.1",
+    "js-yaml": "~3.7.0",
     "mkdirp": "~0.5.1",
-    "csso": "~2.3.1"
+    "sax": "~1.2.1",
+    "stable": "^0.1.5",
+    "whet.extend": "~0.9.9"
```

Comment:
Should I merge this change into this PR branch?
Or will it be automatically merged when this PR branch is merged to master?